components/match2/wikis/dota2/matchgroup_input_custom.lua

```diff
@@ -0,0 +1,675 @@
+---
+-- @Liquipedia
+-- wiki=dota2
+-- page=Module:MatchGroup/Input/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Json = require('Module:Json')
+local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
+local Opponent = require('Module:Opponent')
+local String = require('Module:StringUtils')
+local Table = require('Module:Table')
+local Variables = require('Module:Variables')
+local Template = require('Module:Template')
+local HeroNames = mw.loadData('Module:HeroNames')
+
+local MatchGroupInput = Lua.import('Module:MatchGroup/Input', {requireDevIfEnabled = true})
+
+local _STATUS_SCORE = 'S'
+local _STATUS_DRAW = 'D'
+local _STATUS_DEFAULT_WIN = 'W'
+local _STATUS_FORFEIT = 'FF'
+local _STATUS_DISQUALIFIED = 'DQ'
+local _STATUS_DEFAULT_LOSS = 'L'
+local _ALLOWED_STATUSES = {
+	_STATUS_DRAW,
+	_STATUS_DEFAULT_WIN,
+	_STATUS_FORFEIT,
+	_STATUS_DISQUALIFIED,
+	_STATUS_DEFAULT_LOSS,
+}
+local _MAX_NUM_OPPONENTS = 2
+local _MAX_NUM_PLAYERS = 10
+local _MAX_NUM_GAMES = 7
+local _DEFAULT_BESTOF = 3
+local _DEFAULT_MODE = Opponent.team
```

Comment:
```suggestion
local _DEFAULT_MODE = 'team'
```
Mode 'Team' and Opponent.team are two independent values that happen to have the value for now